src/utils/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const reviews = axios.create({
+    baseURL: "https://nc-games-2enb.onrender.com/api/reviews"
+})
+
+
+export const fetchReviews = () => {
+    return reviews
+    .get(`/`)
+    .then(({ data }) => {
+        return data.reviews;
+    })
+    .catch((error) => {
```

Comment:
This `.catch` block should be chained onto its invocation in the `useEffect`, rather than here, as that's the end of the promise chain.